### PR TITLE
Creates NavBar

### PR DESCRIPTION
**Pull request by seemavora**

### CODE REVIEW

**Review by Jellyfish25 — APPROVED** [review]
looks good

**Review by tritinpho — APPROVED** [review]
Looks great!

File: studyhelper/src/Pages/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { menuItems } from "./menuItems"
+import './NavBar.css'
+import logo from "./Images/ScribeLogo.png";
+
+console.log(logo);
```
Comment (ysaito-2000) [inline studyhelper/src/Pages/components/NavBar/NavBar.js:6]: should prob remove console.log before pushing

File: studyhelper/src/Pages/Homepage/Homepage.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import React from "react";
+import NavBar from "../components/NavBar/NavBar";
+import "./Homepage.css";
 
 export default function Homepage() {
+    console.log(NavBar)
```
Comment (seemavora) [inline studyhelper/src/Pages/Homepage/Homepage.js:6]: remove

**Review by ysaito-2000 — APPROVED** [review]
looks good

**Pull request merged**